index.html

```diff
@@ -186,31 +197,272 @@ <h4>
-            [=app-region=] CSS property has not been implemented in any user
-            agent, so it is at risk. Note that some user agents use the
-            non-standard CSS property <dfn data-export=""
-            data-lt="-webkit-app-region">`-webkit-app-region`</dfn> for the same
-            purpose.
+        <p class="issue" title="At risk feature">
+          [=app-region=] CSS property has not been implemented in any user
+          agent, so it is at risk. Note that some user agents use the
+          non-standard CSS property <dfn data-export="" data-lt=
+          "-webkit-app-region">`-webkit-app-region`</dfn> for the same purpose.
+        </p>
+        <p>
+          The <dfn data-export="" data-lt="app-region">`app-region`</dfn>
+          property can be used to define with CSS which regions or elements in
+          for example a title bar are draggable.
+        </p>
+        <ul>
+          <li>To enable dragging an element, shall be set to `drag`.
+          </li>
+          <li>To disable dragging an element, shall be set to `no-drag`.
+          </li>
+        </ul>
+      </section>
+    </section>
+    <section>
+      <h2>
+        <code><dfn>tab_strip</dfn></code> member
+      </h2>
+      <p>
+        The `tab_strip` member of the <a data-cite=
+        "appmanifest#web-application-manifest">Web Application Manifest</a> is
+        an <a data-cite="appmanifest#dfn-object">object</a> that contains
+        information about how the application is intended to behave in the
+        [=display mode/tabbed=] display mode. It has the following members:
+      </p>
+      <ul>
+        <li>[=tab_strip/home_tab=]
+        </li>
+        <li>[=tab_strip/new_tab_button=]
+        </li>
+      </ul>
+      <p class="issue" title="Missing exports from APPMANIFEST">
+        "manifest URL" and "start URL" need to be exported from APPMANIFEST.
+        See <a href=
+        "https://github.com/w3c/manifest/pull/1112">w3c/manifest#1112</a>.
+      </p>
+      <section>
+        <h3>
+          <code><dfn>home_tab</dfn></code> member
+        </h3>
+        <p>
+          The `home_tab` member of the [=tab_strip=] object is an ordered map
+          that contains information about a specially marked [=application
+          context=] that, if it exists, is in every app window. This
+          application context has special properties compared to other
+          application contexts, such as restricted navigation and it SHOULD
+          have a different appearance.
+        </p>
+        <p>
+          A <dfn>home tab context</dfn> is the [=application context=] in which
+          these special properties are applied.
+        </p>
+        <aside class="note">
+          The home tab is intended to serve as a top level menu for the app.
+        </aside>
+        <p>
+          If the [=tab_strip/home_tab=] member is not present, then the
+          application is not intended to have a home tab context, otherwise it
```

Comment:
Should 'home tab context' here be a link? Same for line 301

Reply:
Done.